src/prompt-input/internal.tsx

```diff
@@ -35,7 +38,7 @@ const InternalPromptInput = React.forwardRef(
       disableActionButton,
       disableBrowserAutocorrect,
       disabled,
-      maxRows = 3,
+      maxRows,
```

Comment:
this line could be updated too: https://github.com/cloudscape-design/components/pull/3320/files#diff-6edc11baca04a63269cef5814df4a3c0fd6584a2e1651319e463040a07dd5d53R152